docs/local_install_skaffold.md

```diff
@@ -1,51 +1,280 @@
+
+If you wish to comfortably run Substra, it is advised to have:
+
+- At least 30 Go of free space but **50** would really be better!
+- **8 Go of RAM** for Kubernetes/Minikube! Be warned, 4 Go won't work. 
+
+### Software requirements
+
+> Note: Please **always** refer to the package provider website before installing any software!
+
+> Note: This setup has been tested on Ubuntu 19.10 & 18.04.3, Linux Mint 19 Cinnamon (4.15.0-45-generic Kernel)
+
+Substra deployment is orchestrated by `Kubernetes` and `Minikube` is a great tool for your local Kubernetes deployments.
+
+`Helm` is a Kubernetes package manager that helps install and manage Kubernetes applications. `Tiller` runs inside the Kubernetes Cluster and manages releases of your charts. Helm has two parts : helm (client) & tiller (server). Charts are Helm packages that contains a package description (`Chart.yaml`) and one or more templates which contain Kubernetes manifest files. Charts can be stored locally or on a distant repository.
+
+`Skaffold` is a program working on top of the Kubernetes configuration that will operate the deployment for you. It relies on the `skaffold.yaml` files that you will find at the root of each repositories.
+
+#### [1. Kubernetes](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+
+##### Mac
+
+First of all, download the `Docker desktop` installer from https://www.docker.com/products/docker-desktop. You'll have to create an account there to do so. Then run it to install Docker on your machine.
 
-This guide will help you run the Substra platform on your machine in development mode, with a 2 organisations setup.
+Once installed, launch Docker and open its "preferences" panel. In the Kubernetes tab, check the `Enable Kubernetes` checkbox.
+
+Kubernetes will take a while to launch the first time, but once it is done, you can move on to configuring.
+
+If you don't have `homebrew` installed, please install it first with the following command:
+
+```sh
+/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+```
+
+##### Ubuntu
 
-- [Setup](#setup)
-  - [1. Get the source code](#1-get-the-source-code)
-  - [2. Install Docker and Kubernetes](#2-install-docker-and-kubernetes)
-  - [3. Configure Kubernetes](#3-configure-kubernetes)
-  - [4. Install skaffold](#4-install-skaffold)
-- [Running the platform](#running-the-platform)
-- [Login, password and urls](#login-password-and-urls)
+Please use **[Kubernetes v1.15](https://kubernetes.io/docs/tasks/tools/install-kubectl/)** (client & server):
 
-## Setup
+```sh
+curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.15.0/bin/linux/amd64/kubectl
+chmod +x ./kubectl
+sudo mv ./kubectl /usr/local/bin/kubectl
+kubectl version --client
+```
+
+#### [2. Minikube](https://minikube.sigs.k8s.io/docs/start/)
+
+```sh
+# Mac
+brew install minikube
```

Comment:
Are you sure we want to direct mac users to minikube ? The bundled k8s is much easier to use.